src/main/java/TaskList.java

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+
+/**
+ * Class to represent the list of tasks that
+ * are being tracked.
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+    private int size;
+
+    /**
+     * Constructor.
+     * @param tasks tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+        this.size = this.tasks.size();
+    }
+
+    /**
+     * Function to add a task into the task list.
+     * @param t the task to be added.
+     */
+    public void addTask(Task t) {
+        this.tasks.add(t);
```

Comment:
There is no need for you to use "this." because this refers to this instance. You only need to use "this" is when you have a class variable and method argument having the same name. "this" will help the compiler differentiate which variable you are referring to.